apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -2120,13 +2140,20 @@ static void libc_stdio_meminstream_tc(void)
 */
 static void libc_stdio_memoutstream_tc(void)
 {
-	FAR char buf[STDIO_BUFLEN];
+	char buffer[20];
+	char *buf = STREAM_TEST_CONTENTS;
 
 	struct lib_memoutstream_s memoutstream;
 
-	lib_memoutstream((FAR struct lib_memoutstream_s *)&memoutstream, buf, STDIO_BUFLEN);
-	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buffer, (FAR char *)(buf));
-	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buflen, (STDIO_BUFLEN - 1));	/* Save space for null terminator, hence checing with (STDIO_BUFLEN-1)*/
+	/* Check with valid BUFLEN */
+
+	lib_memoutstream((FAR struct lib_memoutstream_s *)&memoutstream, buffer, STDIO_BUFLEN);
```

Comment:
3rd parameter is size of the buffer, it should be same with the length of buffer here.
So please change this or 20. I think second one is better

Reply:
Additionally, this is definitely incorrect I think.
Because put function try to put a character to buffer when nput is smaller than buflen.
if (this->nput < mthis->buflen) {
	mthis->buffer[this->nput] = ch;
	...
}
In this case, buflen is set as STDIO_BUFLEN(64) and real length of buffer is 20.
put function writes something to wrong region from 20 tries
Although you didn't try this here, please fix it

Reply:
ok, will do that